README.md

```diff
@@ -54,6 +56,17 @@ In addition to standards provided by [default](https://github.com/squizlabs/PHP_
 * WordPress-VIP
 * Zend
 
+### Custom Standards
+
+You can also include a custom phpcodesniffer standard configuration file. Add
+the file to the engine config standard section as a relative path:
+
+    engines:
+      phpcodesniffer:
+        enabled: true
+        config:
+          standard: "phpcs.ruleset.xml"
```

Comment:
Does this information also live on our docs site? If so, should we link there to avoid going out of sync?